cmd/node_agent/main.go

```diff
@@ -41,15 +41,15 @@ func init() {
 
 	flags.StringVar(&naConfig.ServiceIdentityOrg, "org", "", "Organization for the cert")
 	flags.IntVar(&naConfig.RSAKeySize, "key-size", 1024, "Size of generated private key")
-	flags.StringVar(&naConfig.CertChainFile, "cert-chain",
-		"/etc/certs/cert-chain.pem", "Node Agent identity cert file")
-	flags.StringVar(&naConfig.KeyFile,
-		"key", "/etc/certs/key.pem", "Node identity private key file")
 	flags.StringVar(&naConfig.IstioCAAddress,
 		"ca-address", "istio-ca:8060", "Istio CA address")
-	flags.StringVar(&naConfig.RootCACertFile, "root-cert",
-		"/etc/certs/root-cert.pem", "Root Certificate file")
 	flags.StringVar(&naConfig.Env, "env", "onprem", "Node Environment : onprem | gcp | aws")
+	flags.StringVar(&naConfig.PlatformConfig.CertChainFile, "cert-chain",
```

Comment:
just a thought: is it possible to enable a custom flagset for different platform? Because for other environments, the node agent bootstrap tokens may not be cert/key pair.